src/egon/data/datasets/power_plants/mastr.py

```diff
@@ -313,6 +313,23 @@ def voltage_levels(p: float) -> int:
         units = units.loc[units.Land == "Deutschland"]
         print(f"    {len_old-len(units)} units outside of Germany dropped...")
 
+        # drop units installed after reference date (eGon2021 scenario)
+        len_old = len(units)
+        ts = pd.Timestamp("2022-01-01 00:00:00")
```

Comment:
Except maybe move `"2022-01-01 00:00:00"` into the `datasets.yml`?

Reply:
Good point, done.